students/BrianE/lesson01/inventory_management/ElectricAppliancesClass.py

```diff
@@ -0,0 +1,18 @@
+""" ElectricAppliances Class """
+
+
```

Comment:
Since we are running our tests from the inventory_management folder, you'll want to modify your imports to be pulled from there. Ex:
from inventory_management.InventoryClass import Inventory